core/include/core/transform_store.hpp

```diff
@@ -78,11 +131,24 @@ namespace detray
          *
          * @note in general can throw an exception
          */
-        void add_contextual_transforms(const context & /*ctx*/, storage &&trfs) noexcept(false)
```

Comment:
The naming here was "add" as of adding to a new context, but I agree we can call this "set".